### PR TITLE
Update "multiple schedulers" example

### PR DESCRIPTION
**Pull request by chirangaalwis**
**Description:**
Update multiple scheduler deployment example to align with changes introduced via `v1.22`. Addresses https://github.com/kubernetes/website/issues/29342.

**Test Environment:**
```
Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.1", GitCommit:"5e58841cce77d4bc13713ad2b91fa0d961e69192", GitTreeState:"clean", BuildDate:"2021-05-12T14:18:45Z", GoVersion:"go1.16.4", Compiler:"gc", Platform:"darwin/arm64"}
Server Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.1", GitCommit:"632ed300f2c34f6d6d15ca4cef3d3c7073412212", GitTreeState:"clean", BuildDate:"2021-08-20T00:45:19Z", GoVersion:"go1.16.7", Compiler:"gc", Platform:"linux/amd64"}
```
Furthermore, used the default `kube-scheduler` image `k8s.gcr.io/kube-scheduler:v1.22.0` for testing.

### CODE REVIEW

File: content/en/examples/admin/sched/my-scheduler.yaml [hunk 1]
```diff
@@ -30,6 +30,22 @@ roleRef:
   name: system:volume-scheduler
   apiGroup: rbac.authorization.k8s.io
 ---
+
+apiVersion: v1
+data:
```
Comment (sftim) [inline content/en/examples/admin/sched/my-scheduler.yaml:35]: nit: I'd put this after `kind` so it's more obvious that this is a ConfigMap being defined.

File: content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md [hunk 2]
```diff
@@ -110,11 +119,11 @@ pod in this list.
 
 To run multiple-scheduler with leader election enabled, you must do the following:
 
-First, update the following fields in your YAML file:
+First, update the following fields in the Scheduler Configuration of your YAML file:
```
Comment (sftim) [inline content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:122]:
```suggestion
First, update the scheduler configuration (earlier you applied a
`my-scheduler.yaml` file that included a ConfigMap named `my-scheduler-config`; the
scheduler configuration is in YAML format, within the `data` field of that ConfigMap).

The fields to change in the inner KubeSchedulerConfiguration are:
```

File: content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md [hunk 3]
```diff
@@ -71,15 +71,24 @@ config. Save it as `my-scheduler.yaml`:
 
 {{< codenew file="admin/sched/my-scheduler.yaml" >}}
 
-An important thing to note here is that the name of the scheduler specified as an
-argument to the scheduler command in the container spec should be unique. This is the name that is matched against the value of the optional `spec.schedulerName` on pods, to determine whether this scheduler is responsible for scheduling a particular pod.
+In the above manifest, we have used a [Scheduler Configuration](/docs/reference/scheduling/config/)
+to customize the behavior of your scheduler implementation. This configuration has been passed to
```
Comment (alculquicondor) [inline content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:75]: mention the ConfigMap and the volume mount

File: content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md [hunk 4]
```diff
@@ -168,8 +181,8 @@ scheduler in that pod spec. Let's look at three examples.
   {{< codenew file="admin/sched/pod3.yaml" >}}
 
   In this case, we specify that this pod should be scheduled using the scheduler that we
-  deployed - `my-scheduler`. Note that the value of `spec.schedulerName` should match the name supplied to the scheduler
-  command as an argument in the deployment config for the scheduler.
+  deployed - `my-scheduler`. Note that the value of `spec.schedulerName` should match the name supplied for the scheduler
+  in the `schedulerName` field of the mapping `KubeSchedulerProfile` of the scheduler.
```
Comment (alculquicondor) [inline content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:185]:
```suggestion
  in the `schedulerName` field of the mapping `KubeSchedulerProfile`.
```

File: content/en/examples/admin/sched/my-scheduler.yaml [hunk 5]
```diff
@@ -74,7 +88,14 @@ spec:
...
+        volumeMounts:
+          - name: config-volume
+            mountPath: /etc/kubernetes/my-scheduler
       hostNetwork: false
       hostPID: false
-      volumes: []
+      volumes:
+        - name: config-volume
+          configMap:
+            # Provide the name of the ConfigMap containing the files you want
```
Comment (alculquicondor) [inline content/en/examples/admin/sched/my-scheduler.yaml:99]: The comment seems unnecessary

File: content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md [hunk 6]
```diff
@@ -71,15 +71,26 @@ config. Save it as `my-scheduler.yaml`:
 
 {{< codenew file="admin/sched/my-scheduler.yaml" >}}
 
-An important thing to note here is that the name of the scheduler specified as an
-argument to the scheduler command in the container spec should be unique. This is the name that is matched against the value of the optional `spec.schedulerName` on pods, to determine whether this scheduler is responsible for scheduling a particular pod.
-
-Note also that we created a dedicated service account `my-scheduler` and bind the cluster role
+In the above manifest, we have used a [Scheduler Configuration](/docs/reference/scheduling/config/)
```
Comment (reylejano) [inline content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:74]:
```suggestion
In the above manifest, you use a [Scheduler Configuration](/docs/reference/scheduling/config/)
```
Under [Content best practices](https://kubernetes.io/docs/contribute/style/style-guide/#content-best-practices) in the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/) suggests to use present tense and to address the reader as you not we

File: content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md [hunk 7]
```diff
@@ -71,15 +71,26 @@ config. Save it as `my-scheduler.yaml`:
...
+to customize the behavior of your scheduler implementation. This configuration has been passed to
+the `kube-scheduler` during its initialization via `--config` command line argument. The configuration
+file has been embedded within the ConfigMap `my-scheduler-config` which has been injected to the scheduler
+Deployment via a volume.
```
Comment (reylejano) [inline content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:78]:
```suggestion
the `kube-scheduler` during initialization with the `--config` option. The `my-scheduler-config` ConfigMap stores the configuration file. The Pod of the`my-scheduler` Deployment mounts the `my-scheduler-config` ConfigMap as a volume.
```
nit: for tense

**Comment by chirangaalwis** [comment]
@reylejano please find the suggestions integrated into the PR. Appreciate if this could be reviewed and approved.

File: content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md [hunk 8]
```diff
@@ -71,15 +71,24 @@ config. Save it as `my-scheduler.yaml`:
...
+the `kube-scheduler` during initialization with the `--config` option. The `my-scheduler-config` ConfigMap stores the configuration file. The Pod of the`my-scheduler` Deployment mounts the `my-scheduler-config` ConfigMap as a volume.
 
-Note also that we created a dedicated service account `my-scheduler` and bind the cluster role
+In the aforementioned Scheduler Configuration, your scheduler implementation has been represented via
+a [KubeSchedulerProfile](/docs/reference/config-api/kube-scheduler-config.v1beta2/#kubescheduler-config-k8s-io-v1beta2-KubeSchedulerProfile).
+An important thing to note here is that the name of the scheduler specified via `schedulerName` field of the defined `KubeSchedulerProfile`,
+should be unique among all schedulers running in the cluster. This is the name that is matched against the value of the optional `spec.schedulerName` on pods, to determine whether
+this scheduler is responsible for scheduling a particular pod.
+
+Also, note that we created a dedicated service account `my-scheduler` and bound the cluster role
```
Comment (reylejano) [inline content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:84]:
```suggestion
In the aforementioned Scheduler Configuration, your scheduler implementation is represented via
a [KubeSchedulerProfile](/docs/reference/config-api/kube-scheduler-config.v1beta2/#kubescheduler-config-k8s-io-v1beta2-KubeSchedulerProfile).
{{< note >}}
To determine if a scheduler is responsible for scheduling a specific Pod, the `spec.schedulerName` field in a 
PodTemplate or Pod manifest must match the `schedulerName` field of the `KubeSchedulerProfile`.
All schedulers running in the cluster must have unique names.
{{< /note >}}

Also, note that you create a dedicated service account `my-scheduler` and bind the ClusterRole
```

Maybe use the note shortcode to highlight the content

File: content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md [hunk 9]
```diff
@@ -71,15 +71,24 @@ config. Save it as `my-scheduler.yaml`:
 
 {{< codenew file="admin/sched/my-scheduler.yaml" >}}
 
-An important thing to note here is that the name of the scheduler specified as an
-argument to the scheduler command in the container spec should be unique. This is the name that is matched against the value of the optional `spec.schedulerName` on pods, to determine whether this scheduler is responsible for scheduling a particular pod.
+In the above manifest, you use a [Scheduler Configuration](/docs/reference/scheduling/config/)
```
Comment (reylejano) [inline content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:74]: Is it better to refer to Scheduler Configuration as KubeSchedulerConfiguration?
If Scheduler Configuration is not an API object then it should be lower case

File: content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md [hunk 10]
```diff
@@ -110,11 +119,15 @@ pod in this list.
 
 To run multiple-scheduler with leader election enabled, you must do the following:
 
-First, update the following fields in your YAML file:
+First, update the scheduler configuration (earlier you applied a
```
Comment (reylejano) [inline content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:122]: See my [comment about scheduler configuration](https://github.com/kubernetes/website/pull/30107/files#r740745459).
In either case, we should keep the case the same on this page as 
>Scheduler Configuration

is upper camel case in other parts of the page

File: content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md [hunk 11]
```diff
@@ -110,11 +119,15 @@ pod in this list.
...
+`my-scheduler.yaml` file that included a ConfigMap named `my-scheduler-config`; the
+scheduler configuration is in YAML format, within the `data` field of that ConfigMap).
+
+The fields to change in the inner KubeSchedulerConfiguration are:
```
Comment (reylejano) [inline content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:126]:
```suggestion
Update the following fields for the KubeSchedulerConfiguration in the `my-scheduler-config` ConfigMap in your YAML file:
```

File: content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md (hunk 9 above)
Comment (chirangaalwis) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:74]: @reylejano IMO, it is better to use `KubeSchedulerConfiguration` as this is a API resource type although it is not a server resource type.

@alculquicondor @damemi please correct me if I am wrong.

Comment (chirangaalwis) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:74]: @reylejano IMO, it is better to use `KubeSchedulerConfiguration` as this avoids any confusion regarding the resource type (this is an API resource type although it is not a server resource type). @alculquicondor @damemi WDYT?

Comment (damemi) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:74]: I usually try to use `KubeSchedulerConfiguration` to show that I am specifically talking about this file/API object (and not, for example, other types of config like flags/policy/etc...)

Comment (alculquicondor) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:74]: `KubeSchedulerConfiguration` sgtm

Comment (chirangaalwis) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:74]: @damemi @alculquicondor ack. Will update.

Comment (chirangaalwis) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md]: @reylejano updated in https://github.com/kubernetes/website/pull/30107/commits/7449bb36a0eaa7f2983fa8fae005e00f06496fa8.

File: content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md (hunk 10 above)
Comment (chirangaalwis) [reply content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md]: @reylejano already updated with https://github.com/kubernetes/website/pull/30107/commits/c5cb03ab13e4e101875e03a61689cad458ec782f.

**Comment by chirangaalwis** [comment]
@reylejano updated based on review comments. Please review and approve.

**Comment by reylejano** [comment]
Thank you so much for the updates
/lgtm

**Pull request merged**